bitnami/mongodb/values-production.yaml

```diff
@@ -188,15 +192,23 @@ initdbScripts: {}
 ## extraFlags:
 ##  - "--wiredTigerCacheSizeGB=2"
 ##
-extraFlags: []
+#extraFlags: []
+extraFlags:
+  - "--tlsMode=requireTLS"
+  - "--tlsCertificateKeyFile=/certs/mongodb.pem"
+  - "--tlsCAFile=/certs/mongodb-ca-cert"
```

Comment:
We should not change the default parameters because it will require a major change